tests/lib/cli-engine/formatters/stylish.js

```diff
@@ -80,12 +38,14 @@ describe("formatter:stylish", () => {
 		];
 
 		it("should not return message", () => {
-			const result = formatter(code);
+			const result = util.stripVTControlCharacters(formatter(code));
 
 			assert.strictEqual(result, "");
-			assert.strictEqual(chalkStub.reset.callCount, 0);
-			assert.strictEqual(chalkStub.yellow.bold.callCount, 0);
-			assert.strictEqual(chalkStub.red.bold.callCount, 0);
+			/*
+			 * assert.strictEqual(chalkStub.reset.callCount, 0);
+			 * assert.strictEqual(chalkStub.yellow.bold.callCount, 0);
+			 * assert.strictEqual(chalkStub.red.bold.callCount, 0);
+			 */
```

Comment:
To preserve the `callCount` assertion:

```suggestion
			 assert.strictEqual(util.styleText.callCount, 0);
```

or more idiomatically:

```suggestion
			 assert(util.styleText.notCalled);
```